libraries/lib-music-information-retrieval/PowerSpectrumGetter.cpp

```diff
@@ -17,22 +17,34 @@ namespace MIR
 PowerSpectrumGetter::PowerSpectrumGetter(int fftSize)
     : mFftSize { fftSize }
     , mSetup { pffft_new_setup(fftSize, PFFFT_REAL) }
-    , mWork(fftSize)
+    , mData { reinterpret_cast<float*>(
+         pffft_aligned_malloc(fftSize * sizeof(float))) }
+    , mWork { reinterpret_cast<float*>(
+         pffft_aligned_malloc(fftSize * sizeof(float))) }
```

Comment:
So I understand that this change in the construction was necessary to avoid the crash -- the allocation in the `vector<float>` construction was not guaranteed to be aligned.

But you do not make similar changes in the unit tests.

Could the unit tests have intermittent crashes too?

Reply:
PFFFT isn't exposed by the library, so no changes are needed in the UTs.
I haven't experience a crash in the unit tests. It appears to happen seldom. So far I've only been able to reproduce it intermittently with the wav file attached to the ticket.

Reply:
When running the benchmarking test in debug, I actually fail the assert in pffft that my data is aligned. I normally run it in release, I guess that's why I haven't noticed it earlier.